scout/commands/export/variant.py

```diff
@@ -105,6 +105,41 @@ def verified(collaborator, test, outpath=None):
     return written_files
 
 
+@click.command("managed", short_help="Export managed variants")
+@click.option(
+    "-c",
+    "--collaborator",
+    help="Specify what collaborator to export variants from. Defaults to all variants.",
+)
+@click.option(
```

Comment:
This could be treated as en eval type of input, so you force the user to provide the right build values from a list, -> https://click.palletsprojects.com/en/8.1.x/options/#choice-options . I know build is not used, but since you added it..